File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/forloop.java
import java.util.Scanner;

public class question4 {

    public static void main(String[] args) {

        try (/*
              * Your class should be named Solution.
              * Read input as specified in the question.
              * Print output as specified in the question.
              */
                Scanner read = new Scanner(System.in)) {
            int count = read.nextInt();
            System.out.println(F(count));
        }

    }

    /*
     * static int printFibonacci(int count){
     * if(countif(count>0){
     * n3 = n1 + n2;
     * n1 = n2;
     * n2 = n3;
     * printFibonacci(count-1);
     * 
     * }
     * return n3;
     * }
     */
    public static int F(int n) {
        if (n == 1 || n == 2) {
            return 1;
        }

        if (n > 2) {
            return F(n - 1) + F(n - 2);
        }
        return 0;
    }
}

File: SecondYear/seeeyaa_PariGupta_2226CSE(AI)1096_2/Week_1/ForLoop.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
        /*
        *  Write your code here. 
         *  Read input as specified in the question.
         *  Print output as specified in the question.
        */
        int n,a=1,b=1,c;
        cin>>n;
        if(n==1) cout<<a;
        else if(n==2)  cout<<b;
        else
        {
             for(int i=3;i<=n;i++)
        {
         c=a+b;
         a=b;
         b=c;
        }
        cout<<c;
        }
        
        

        return 0;
       
        
        


}